substrate/primitives/core/src/lib.rs

```diff
@@ -46,11 +46,14 @@ pub use sp_debug_derive::RuntimeDebug;
 #[cfg(feature = "serde")]
 pub use impl_serde::serialize as bytes;
 
+// https://github.com/rust-lang/rust/issues/85388
 #[cfg(feature = "full_crypto")]
-pub mod hashing;
+#[deprecated(
+	since = "21.0.0",
+	note = "hashing re-export is scheduled for removal, please use `sp-crypto-hashing` instead"
```

Comment:
Maybe we should give some removal date here? (+6 months?)

Reply:
https://github.com/paritytech/polkadot-sdk/blob/caa987d26d14bbdbfbc9be08a1da4c2b117fcaa3/docs/contributor/DEPRECATION_CHECKLIST.md#hard-deprecate-by-adding-a-warning-message